meta-lmp-base/recipes-support/apps-early-start/apps-early-start/lmp-apps-early-start

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh -e
+
+if [ -f /var/sota/sql.db ] ; then
```

Comment:
lmp-device-auto-register has `ConditionPathExists=!/var/sota/sql.db`, so this condition will never really be false when started from systemd.

Do we expect people to manually call this script? Otherwise we can probably drop this and reduce one if block.

Reply:
Actually, since it is before we might have it executed, but would need to check (on the dependency chain lmp-device-auto-register.service is never executed when /var/sota/sql.db exists, so this job might not be executed as consequence).

To be safe we could also have the same condition and remove this if block from the script.

Reply:
k, i think i've got this fixed in the force push